Firestore/Source/Remote/FSTSerializerBeta.h

```diff
@@ -84,9 +84,6 @@ NS_ASSUME_NONNULL_BEGIN
 - (GPBTimestamp *)encodedVersion:(const model::SnapshotVersion &)version;
 - (model::SnapshotVersion)decodedVersion:(GPBTimestamp *)version;
 
-/** Returns the database ID, such as `projects/{project id}/databases/{database_id}`. */
```

Comment:
Removing these now makes me vaguely uneasy because it will make it harder to perform additional audits of the serializer at the point when we're trying to cut over.